src/components/index/Card.tsx

```diff
@@ -0,0 +1,46 @@
+import { clsx } from "clsx";
+import Image from "next/image";
+
+interface CardProps {
+  readonly title: string;
+  readonly description: string;
+  readonly imageSrc?: string;
+  readonly className?: string;
+  readonly isPriority?: boolean;
+}
+
+export function Card({
+  title,
+  description,
+  imageSrc,
+  className,
+  isPriority = false,
+}: CardProps) {
+  return (
+    <div
+      className={clsx(
+        "w-full overflow-hidden rounded-2xl border border-gray-100/20 p-8 transition hover:bg-indigo-100 dark:hover:bg-gray-900/80",
+        "flex flex-col items-center justify-start",
+        className,
+      )}
+    >
+      <h3 className="mb-4 w-full bg-clip-text text-2xl font-bold dark:text-white sm:text-3xl md:text-4xl">
+        {title}
+      </h3>
+      <p className="mx-auto mt-4 text-lg text-slate-600 dark:text-slate-400">
+        {description}
+      </p>
+
+      {imageSrc ? (
+        <Image
+          className="translate-y-1/3 scale-125"
+          src={imageSrc}
+          alt="Picture of the author"
+          width={500}
```

Comment:
Why are we enforcing these values? I think we should pass here an object with image config as a prop instead of just src